.gitignore

```diff
@@ -4,6 +4,8 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
I'm not sure about yarn.lock, but don't we generally want to have package-lock.json be checked into git so it can track which version of the packages we're all on? Otherwise our package-locks might be different on different environments & have different dependencies installed.